### PR TITLE
Add verbose logging to prune-recursion

### PR DESCRIPTION
**Pull request by andrie**

### CODE REVIEW

File: R/pkg.R [hunk 1]
```diff
@@ -185,11 +185,18 @@ getPackageRecords <- function(pkgNames,
                               missing.package = error_not_installed,
                               check.lockfile = FALSE,
                               fallback.ok = FALSE,
-                              .visited.packages = new.env(parent = emptyenv()))
+                              prune.search.tree = FALSE,
```
Comment (aronatkins) [inline R/pkg.R:188]: These additional arguments are not added to the recursive call.
Additionally, `.recursion.level` is always 1.

File: R/pkg.R [hunk 2]
```diff
@@ -268,9 +275,22 @@ getPackageRecords <- function(pkgNames,
   # Remove any null records
   allRecords <- dropNull(allRecords)
 
+  if (.recursion.level == 1) {
+    .nnn <- length(allRecords)
+    .iii <- 1
+  }
+  if (prune.search.tree) {
```
Comment (aronatkins) [inline R/pkg.R:282]: This conditional code is obsolete; the use of `.visited.packages` has changed and now contains full records.

**Comment by aronatkins** [comment]
CC @kevinushey

File: R/pkg.R (hunk 1 above)
Comment (andrie) [reply R/pkg.R:188]: Thank you. I'll fix.

File: R/pkg.R (hunk 2 above)
Comment (andrie) [reply R/pkg.R:282]: Thank you.  I'll fix.

**Comment by aronatkins** [comment]
This has been merged as https://github.com/rstudio/packrat/pull/616.

**Pull request closed without merging**